engine/class_modules/apl/retribution_apl.inc

```diff
@@ -12,22 +12,24 @@ actions.precombat+=/variable,name=trinket_2_sync,op=setif,value=1,value_else=0.5
 actions.precombat+=/variable,name=trinket_priority,op=setif,value=2,value_else=1,condition=!trinket.1.has_use_buff&trinket.2.has_use_buff|trinket.2.has_use_buff&((trinket.2.cooldown.duration%trinket.2.proc.any_dps.duration)*(1.5+trinket.2.has_buff.strength)*(variable.trinket_2_sync))>((trinket.1.cooldown.duration%trinket.1.proc.any_dps.duration)*(1.5+trinket.1.has_buff.strength)*(variable.trinket_1_sync))
 actions.precombat+=/variable,name=trinket_1_buffs,value=trinket.1.has_buff.strength|trinket.1.has_buff.mastery|trinket.1.has_buff.versatility|trinket.1.has_buff.haste|trinket.1.has_buff.crit
 actions.precombat+=/variable,name=trinket_2_buffs,value=trinket.2.has_buff.strength|trinket.2.has_buff.mastery|trinket.2.has_buff.versatility|trinket.2.has_buff.haste|trinket.2.has_buff.crit
+actions.precombat+=/variable,name=specified_trinket,value=equipped.blazebinders_hoof
```

Comment:
Might be better to specify trinket slot. similar to how evoker has done it. This will allow you to leave the logic in for the secondary trinket if necessary. 

`  precombat->add_action( "variable,name=trinket_1_manual,value=trinket.1.is.spoils_of_neltharus" );
  precombat->add_action( "variable,name=trinket_2_manual,value=trinket.2.is.spoils_of_neltharus" );`
  
  then flag each trinket slots use with `!variable.trinket_1_manual` or, whatever you name it.